scripts/mail-alarm

```diff
@@ -497,7 +620,8 @@ def main():
         min_priority = 3
 
     charset = other_config.get('mail-charset', 'utf-8')
-    msg = XapiMessage(sys.argv[1])
+    mail_language = get_mail_language(other_config)
+    msg = XapiMessage(sys.argv[1], mail_language)
```

Comment:
I know that it was already there, but I think we should check that `len(sys.argv) > 1` here and log an appropriate error if it is not the case